bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/config/config-descriptions.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:fineoffsetweatherstation:gateway">
+		<parameter name="ip" type="text" required="true">
+			<label>IP address</label>
+		</parameter>
+		<parameter name="port" type="integer" required="true">
+			<label>Port</label>
+			<default>45000</default>
+		</parameter>
+		<parameter name="pollingInterval" type="integer" required="true">
+			<label>Polling period</label>
```

Comment:
Labels should have all words start with a capital.

```suggestion
			<label>Polling Period</label>
```

Also consider if it becomes confusing for textual users when you call the name and the label two very similar things, can they be the same?